ansible/roles/cli/tasks/download_remote_redirect_cli.yml

```diff
@@ -0,0 +1,38 @@
+# Download the cli package from the remote_redirect location to openwhisk_home/bin and unarchive it.
+
+- name: "set the file name to download for linux"
+  set_fact:
+    cli_download_name={{ openwhisk_cli.remote_redirect.package_name }}-{{ openwhisk_cli_tag }}-linux-amd64.tgz
```

Comment:
This is a bit more complicated now, because there will be multiple linux architectures (amd64, s390x).  Is there a reason we can't reuse the existing ansible files for downloading the cli?  It seems configuring Nginx should be a separate problem from downloading a binary for "local" use.